dCommon/dEnums/StringifiedEnum.h

```diff
@@ -6,20 +6,26 @@
 
 namespace StringifiedEnum {
 	template<typename T>
-	const std::string_view ToString(const T e) {
+	constexpr std::string_view ToString(const T& e) {
+		// Check type
+		static_assert(std::is_enum<T>::value, "Not an enum");
```

Comment:
These should have an _v variant so you dont need to do ::value if memory serves me correctly

Reply:
`is_enum_v<T>`